Uno.Gallery/Uno.Gallery.Wasm/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "noImplicitAny": true,
     "outFile": "WasmScripts/UnoGallery.js",
     //"inlineSourceMap": true,
-    "locale": "en-US",
+    // Build:Option 'locale' can only be specified on command line.
+    //"locale": "en-US",
```

Comment:
I couldn't build the Wasm project because of:

`error TS6266: Build:Option 'locale' can only be specified on command line.`

I commented out the locale since I noticed that our other Wasm apps don't have it.